packages/itwinui-css/backstop/tests/file-upload.html

```diff
@@ -659,5 +659,67 @@ <h2>Not wrapping anything</h2>
 
     </section>
 
+    <hr />
+
+    <h2>File uploaded</h2>
+    <section id="demo-file-uploaded">
+
+      <div class="iui-file-upload">
+        <div class="iui-content">
+          <div class="iui-file-uploaded-template">
+            <svg-filetype-pdf
+              class="iui-file-uploaded-template-icon"
+              aria-hidden="true"
+            ></svg-filetype-pdf>
+            <span class="iui-file-uploaded-template-text">
+              <span class="iui-file-uploaded-template-label">File title</span>
+              <span class="iui-file-uploaded-template-description">15KB Wed Jan 11 2023 5:25:59 PM</span>
+            </span>
+            <div class="iui-file-uploaded-template-action">
+              <label class="iui-anchor">
+                <input
+                  class="iui-browse-input"
+                  type="file"
+                  name="files[]"
+                  data-multiple-caption="{count} files selected"
+                  multiple
+                />
+                Replace
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <br />
+
+      <div class="iui-file-upload iui-drag">
```

Comment:
This example doesn't look quite right to me/match the others - the blue background doesn't cover the entire area. Is this what it's supposed to look like?
![image](https://user-images.githubusercontent.com/37936169/214886543-3ceae624-3d20-45d8-b756-df9920e9c594.png)

Reply:
This was due to a change in the background color that I've made and since reverted. This should be fixed now